magic.js

```diff
@@ -0,0 +1,5 @@
+const { Magic } = require('@magic-sdk/admin');
+ 
+const magic = new Magic('sk_test_D247CB3A8247F971');
```

Comment:
For setting up private keys, use environment variable.

Codebase is using [config](https://www.npmjs.com/package/config) lib for it.

Reply:
Sorry, missed this one. 
Btw, I reconfigured the codebase to use `dotenv` lib to load env variables directly into the `process.env` object. 
Keys will go into a `.env` file.

Reply:
Got it.

Reply:
Remove all dependency related to config lib since we are now using dotenv.